homeassistant/components/mqtt/strings.json

```diff
@@ -181,11 +181,13 @@
           "data": {
             "platform": "Type of entity",
             "name": "Entity name",
+            "entity_category": "Entity category",
             "entity_picture": "Entity picture"
           },
           "data_description": {
             "platform": "The type of the entity to configure.",
             "name": "The name of the entity. Leave empty to set it to `None` to [mark it as main feature of the MQTT device](https://www.home-assistant.io/integrations/mqtt/#naming-of-mqtt-entities).",
+            "entity_category": "The category of the entity to configure. Leave empty to set it to `None` to [mark it as main feature of the MQTT device](https://developers.home-assistant.io/docs/core/entity/#registry-properties). An entity with a category will not be exposed to cloud, Alexa, or Google Assistant components, nor included in indirect service calls to devices or areas.",
```

Comment:
The "Leave empty to set it to `None` to [mark it as main feature of the MQTT device]" is not correct, maybe it's a copy-paste-mistake?